### ISSUE TITLE
`background-position` with `calc()` in shorthand form is merged incorrectly

### ISSUE THREAD

**Issue by falsefalse** [jakubpawlowicz/clean-css#422]
This 

```css
select {
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 5px 10px;
  padding-right: 25px;
  line-height: 22px;
  height: 32px;
  background: white calc(100% - 10px) center no-repeat;
  background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAGCAMAAAAi7JTKAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRFAAAA////pdmf3QAAAAJ0Uk5T/wDltzBKAAAAIUlEQVR42mJgYIQBBgZGGAfMYoAywQxkzMgAoxAkQIABAAfmADidlbr2AAAAAElFTkSuQmCC);
}
```

gets optimized to this

```css
select {
  -webkit-appearance: none;
  -moz-appearance: none;
  padding: 5px 25px 5px 10px;
  line-height: 22px;
  height: 32px;
  background: #fff url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAsAAAAGCAMAAAAi7JTKAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAAAZQTFRFAAAA////pdmf3QAAAAJ0Uk5T/wDltzBKAAAAIUlEQVR42mJgYIQBBgZGGAfMYoAywQxkzMgAoxAkQIABAAfmADidlbr2AAAAAElFTkSuQmCC) no-repeat
}
```
Looks like `calc()` isn't handled well.

**Comment by jakubpawlowicz** [comment]
Has just been fixed in 3.0.2!

**Comment by jakubpawlowicz** [comment]
Also backported to 2.2 branch.
